config/example/config.txt

```diff
@@ -77,23 +77,25 @@ KICK_INACTIVE
 ##
 ## default probablity is 1, increase to make that mode more likely to be picked
 ## set to 0 to disable that mode
-PROBABILITY EXTENDED 2
-PROBABILITY NUCLEAR 3
-PROBABILITY ABDUCTION 0
-PROBABILITY CHANGELING 3
-PROBABILITY CULT 4
-PROBABILITY EXTEND-A-TRAITORMONGOUS 5
-PROBABILITY TRAITORCHAN 4
-PROBABILITY TRAITORVAMP 4
-PROBABILITY BLOB 2
+PROBABILITY EXTEND-A-TRAITORMONGOUS 3
+PROBABILITY TRAITORCHAN 3
```

Comment:
"T-Traitor-chan! W-why are you covered in blood?"
"Captain-kun... You wouldn't understand"